### PR TITLE
Check PassiveIP eq External IP, Eximstats to M-M-M-Monster Logs, /var/cpanel/tomcat.options -Xss2m

### PR DESCRIPTION
**Pull request by cPPietila**
3 commits to alter the functionality of SSP.

- Ignore -Xss2m in /var/cpanel/tomcat.options
- Add eximstats_db.sqlite3 to M-M-M-MONSTER LOG
- Add checking that FTP announces correct IP

### CODE REVIEW

File: ssp [hunk 1]
```diff
@@ -2505,8 +2514,28 @@ sub print_ftpserver_info {
 
     my $passivetext = $passiveports ? "enabled - " . ( $fwppactive ? "allowed in iptables" : "not found in iptables" ) : "not enabled";
 
+    my $passiveiptext = "";
+
+    if ($passiveip) {
+        if ( $passiveip ne $EXTERNAL_IP_ADDRESS ) {
+            if ( defined( $CPCONF{'ftpserver'} ) ) {
```
Comment (cPanelChrisD) [inline ssp]: These if blocks should be combined:
if ( defined( $CPCONF{'ftpserver'} ) and ( $passiveip ne $EXTERNAL_IP_ADDRESS ) ) {

File: ssp [hunk 2]
```diff
@@ -2505,8 +2514,28 @@ sub print_ftpserver_info {
...
+                if ( $CPCONF{'ftpserver'} eq 'proftpd' ) {
+                    $passiveiptext = "MasqueradeAddress ( " . $passiveip . " ) doesn't match license IP";
```
Comment (cPanelChrisD) [inline ssp]: Concatenate this text on the end of $passivetext, it's not necessary to put it in a separate $passiveiptext.  6 lines of code can be dropped by doing so, reducing complexity.

**Review by cPanelChrisD — CHANGES_REQUESTED** [review]
Please create and assign JIRA TECH cases to yourself for each issue that you will be addressing in SSP.  For example, you can assign TECH-296 to yourself and mark it as completed (with a link to the matching commit in your personal fork of the repo, please).

This is also going to conflict with the pull request I sent in yesterday.  That merge will additionally begin the requirement that SSP passes perlcritic tests defined in the Makefile with the provided perlcritic profile, in addition to being tidy.  After that pull is merged, you can "make test" to just run the tests, and "make tidy" will run both test and tidy targets.

File: ssp (hunk 1 above)
Comment (davelcpanelnet-zz) [reply ssp]: You probably want to enforce the logic here even further as well:
```
if ( exists $passiveip && $passiveip ne $EXTERNAL_IP_ADDRESS && defined( $CPCONF{'ftpserver'} )
```

**Comment by davelcpanelnet-zz** [comment]
Please make the changes requested and be sure to note the TECH-* for proper visibility.

This pull request will be left open so please feel free to make those updates and even make a few changes should you wish.

**Comment by cPPietila** [comment]
I've gone ahead and made the requested changes.

**Review by cPanelChrisD — APPROVED** [review]
All looks good.  Ready for Dave's review.

**Comment by davelcpanelnet-zz** [comment]
Good to go.

**Pull request merged**